sql/hive/src/test/scala/org/apache/spark/sql/hive/MetastoreDataSourcesSuite.scala

```diff
@@ -1345,6 +1344,17 @@ class MetastoreDataSourcesSuite extends QueryTest with SQLTestUtils with TestHiv
     }
   }
 
+  Seq("orc", "parquet").foreach { format =>
```

Comment:
how about the other built-in formats?

Reply:
which other formats should I include here?

Reply:
`parquet`, `json`, `orc`, `csv`, `text`?